otp/imx6ull-otp/imx6ull-otp.c

```diff
@@ -549,6 +557,48 @@ static int write_sn(const char *sn, int raw)
 }
 
 
+static int read_hwrev(int silent)
+{
+	uint32_t plc1 = *(otp.base + ocotp_gp2);
+
+	// NOTE: keeping (hw_rev - 1) in FUSE bits
```

Comment:
Please use C-style comments
```suggestion
	/* NOTE: keeping (hw_rev - 1) in FUSE bits */
```